src/scripts/deleteRecipe.js

```diff
@@ -3,7 +3,7 @@
  * and changes its value in the delMap (located at index 3 in localstorage) to true.
  * @param {int} id , Id of Recipe to Delete
  */
-export function deleteRecipe (id) {
+ function deleteRecipe (id) {
```

Comment:
Same reason as the previous comment. We import deleteRecipe() into recipecard.js because afaik we dynamically generate the recipe card's delete button, so this should cause a ReferenceError afaik